File: Factory/csharp factory/Mysql.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace csharp_factory
{
    public class Mysql : IDatabase
    {
        public void VeriCek(){
            Console.WriteLine("Mysql'de veri cekme islemi gerceklestirildi.");
        }
        public void VeriEkle(){
            Console.WriteLine("Mysql'de veri ekleme islemi gerceklestirildi.");
        }       
    }
}

File: Factory/python factory/Mysql.py
from IDatabase import IDatabase

class Mysql(IDatabase):
    def veri_ekle(self):
        print("Mysql'de veri ekleme gerceklestirildi.")
    def veri_cek(self):
        print("Mysql'de veri cekme gerceklestirildi.") 